### ISSUE TITLE
[Feature] -borderless option for screenshot command [sf#226]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#250]
**Reported by manuelbi on 2011-05-09 19:09 UTC**
With this option, you would get a screenshot without the border included. This is great if you want to make maps which are continuous; then you don't have to manually crop off the borders.

**Comment by openMSX-import** [comment]
**Commented by bifimsx on 2011-05-10 06:37 UTC**
or add a capure window, allowing to only capture the game screen area.

**Comment by openMSX-import** [comment]
**Commented by sd-snatcher on 2015-01-03 23:03 UTC**
This feature would be of great help when capturing series of screenshots.

**Comment by MBilderbeek** [comment]
> **Commented by bifimsx on 2011-05-10 06:37 UTC** or add a capure window, allowing to only capture the game screen area.

Yes, for mapping purposes, you'd also want to skip the score or status bar areas of the screen. So it would be useful to somehow specify a screenshot area in MSX pixels.

**Comment by MBilderbeek** [comment]
On the other hand: one could also take them as normal and then later batch crop the images in one go with an external tool like ImageMagick.
